gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -7,17 +7,32 @@
 module PostgresHaAdmin
   class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
-    DB_CONNECTED_CHECK_FREQUENCY = 300
+    DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    attr_accessor :failover_attempts, :db_check_frequency, :failover_check_frequency
 
     def initialize(db_yml_file = '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file = '/var/www/miq/vmdb/config/failover_databases.yml',
+                   ha_admin_yml_file = '/var/www/miq/vmdb/config/ha_admin.yml',
                    log_file = '/var/www/miq/vmdb/log/ha_admin.log',
                    environment = 'production')
       @logger = Logger.new(log_file)
       @logger.level = Logger::INFO
       @database_yml = DatabaseYml.new(db_yml_file, environment)
       @failover_db = FailoverDatabases.new(failover_yml_file, @logger)
+      begin
+        ha_admin_yml = YAML.load_file(ha_admin_yml_file)
+      rescue SystemCallError, IOError => err
+        ha_admin_yml = {}
+        @logger.error("#{err.class}: #{err}")
+        @logger.info("File not loaded: #{ha_admin_yml_file}. Default settings for failover will be used.")
+      end
+      @failover_attempts = ha_admin_yml['failover_attempts'] || FAILOVER_ATTEMPTS
+      @db_check_frequency = ha_admin_yml['db_check_frequency'] || DB_CHECK_FREQUENCY
+      @failover_check_frequency = ha_admin_yml['failover_check_frequency'] || FAILOVER_CHECK_FREQUENCY
+      @logger.info("FAILOVER_ATTEMPTS=#{@failover_attempts}"\
+                   "DB_CHECK_FREQUENCY=#{@db_check_frequency}"\
+                   "FAILOVER_CHECK_FREQUENCY=#{@failover_check_frequency}")
```

Comment:
This will not print as expected.  The `\` causes the lines to butt up against each other.  What you have is equivalent to

```ruby
"FAILOVER_ATTEMPTS=#{@failover_attempts}DB_CHECK_FREQUENCY=#{@db_check_frequency}FAILOVER_CHECK_FREQUENCY=#{@failover_check_frequency}"
```

which as you can see will not have spaces between the parts.  This is part of the reason I don't like line continuations, even for strings.